src/elements.jl

```diff
@@ -188,8 +188,9 @@ coord(v::ElemT) where {ElemT<:Union{Point,AbstractVector}} = v
 coord(v::ElemT) where {ElemT<:Union{HRepElement,SymPoint,Ray,Line}} = v.a
 
 const VRepElementContainer = Union{SymPoint, Ray, Line}
+const FixedVRepElementContainer{N,T} = Union{Ray{N,T}, Line{N,T}, SymPoint{N,T}}
 
-function (*)(P::Matrix, v::ElemT) where ElemT<:FixedVRepElement
+function (*)(P::Matrix, v::ElemT) where ElemT<:FixedVRepElementContainer
     if ElemT <: VRepElementContainer
```

Comment:
We can remove this `if else end`, as we should always be in the first case so the `else` is useless.